NitroxServer/GameLogic/EventTriggerer.cs

```diff
@@ -55,12 +57,13 @@ private Timer CreateTimer(double time, StoryEventType eventType, string key)
             Timer timer = new Timer();
             timer.Elapsed += delegate
             {
-                Log.Info("Triggering event type " + eventType.ToString() + " at time " + time.ToString() + " with param " + key.ToString());
+                Log.Info("Triggering event type " + eventType + " at time " + time + " with param " + key);
```

Comment:
Should be `Log.Info($"Trigger event type {event Type}...`.

Reply:
```suggestion
                Log.Info($"Triggering event type {eventType} at time {time} with param {key}");
```